src/components/AfterDetails.js

```diff
@@ -1,33 +1,24 @@
 import React from 'react';
 import {FaCheckCircle} from 'react-icons/fa';
 
-class AfterDetails extends React.Component {
+const AfterDetails = (props) => {
+    const {history, isReturning, title} = props;
+    const mainText = isReturning ? `${title} has been returned` : `You have checked out ${title}`;
+    const buttonText = isReturning ? "Return" : "Check Out";
 
-    constructor() {
-        super();
-        this.state = {
-            complete: false
-        }
-    }
-    render() {
-        const {history, isReturning, title} = this.props;
-        const mainText = isReturning ? `${title} has been returned` : `You have checked out ${title}`;
-        const buttonText = isReturning ? "Return" : "Check Out";
-
-        return (
-            <div style={{display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'flex-start', marginTop: '40px'}}>
-                <FaCheckCircle color="#2AF598" size={100} />
-                <h3>{mainText}</h3>
-                <div style={{display: 'flex', alignItems: 'center'}}>
-                    <button style={{fontSize: 20, margin: 10, width: 250}} onClick={() => history.push('/books')}>Continue to Library</button>
-                    </div>
-                <div style={{display: 'flex', alignItems: 'center'}}>
-                    <button style={{fontSize: 20, margin: 10, width: 250, color: "#2185d0", backgroundColor: "#EAEAEA"}} onClick={() => this.setState({complete: true})}>{`${buttonText} another book`}</button>
-                </div>
-                <p>If you are done close the tab/app</p>
+    return (
+        <div style={{display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'flex-start', marginTop: '40px'}}>
+            <FaCheckCircle color="#2AF598" size={100} />
+            <h3>{mainText}</h3>
+            <div style={{display: 'flex', alignItems: 'center'}}>
+                <button style={{fontSize: 20, margin: 10, width: 250}} onClick={() => history.push('/books')}>Continue to Library</button>
+            </div>
+            <div style={{display: 'flex', alignItems: 'center'}}>
+                <button style={{fontSize: 20, margin: 10, width: 250, color: "#2185d0", backgroundColor: "#EAEAEA"}} onClick={() => {console.log('Planned feature?')}}>{`${buttonText} another book`}</button>
```

Comment:
Should we remove this whole section regarding "Check out another book" button? It was only setting a piece of state, which didn't do anything. Additionally, the button above, "Return to Library" would allow us to have the same functionality.

Reply:
I’m all for removing questionable functionality/code.

Reply:
:+1: Removed